flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroToRowDataConverters.java

```diff
@@ -121,6 +128,12 @@ private static AvroToRowDataConverter createConverter(LogicalType type) {
                 return AvroToRowDataConverters::convertToTime;
             case TIMESTAMP_WITHOUT_TIME_ZONE:
                 return AvroToRowDataConverters::convertToTimestamp;
```

Comment:
IIUC, we should convert Flink SQL TIMESTAMP_NTZ type to AVRO Local Timestamp finally, this is the correct behavior

Reply:
Correct. The incorrect part for Flink SQL TIMESTAMP_NTZ type  is in RowDataToAvroConverters. I just add TIMESTAMP_WITH_LOCAL_TIME_ZONE  handling in the new mapping.

Reply:
This a behavior change that we didn't discuss in dev mail-list, I'm considering to need discuss in mail-list, maybe we should back to mail-list and start a quick discussion, WDYT?

Reply:
In the FLIP, we have proposed the new mapping. Does it cover both the change in AvroToRowDataConverters and RowDataToAvroConverters? 

Avro Timestamp <-> Flink TIMESTAMP_WITH_LOCAL_TIME_ZONE
additionally support Avro LocalTimestamp <-> Flink TIMESTAMP_WITHOUT_TIME_ZONE